person.rb

```diff
@@ -24,4 +25,9 @@ def can_use_service
   def correct_name
     @name
   end
+
+  def rent_book(rental)
+    @rentals.push(rental)
+    rental.person = self
+  end
```

Comment:
- [ ] The same changes I suggested for the Book class applies here as well ☝️

Reply:
Noted.